lib/jekyll-maps/mapbox_api.rb

```diff
@@ -0,0 +1,84 @@
+module Jekyll
+  module Maps
+    class MapboxApi
+      HEAD_END_TAG = %r!</[\s\t]*head>!
+
+      class << self
+        def prepend_api_code(doc)
+          @config = doc.site.config
+          if doc.output =~ HEAD_END_TAG
+            # Insert API code before header's end if this document has one.
+            doc.output.gsub!(HEAD_END_TAG, %(#{api_code}#{Regexp.last_match}))
+          else
+            doc.output.prepend(api_code)
+          end
+        end
+
+        private
+        def api_code
+          <<HTML
+<script type='text/javascript'>
+  #{js_lib_contents}
+</script>
+#{load_mapbox_api}
+#{load_marker_cluster}
+HTML
+        end
+
+        private
+        def load_mapbox_api
+          access_token = @config.fetch("maps", {})
+            .fetch("mapbox", {})
+            .fetch("access_token", "")
+          <<HTML
+
+<script async defer src='https://api.tiles.mapbox.com/mapbox-gl-js/v0.54.0/mapbox-gl.js' onload='#{Jekyll::Maps::MapboxTag::JS_LIB_NAME}.initializeMap("#{access_token}")'></script>
+<link async defer href='https://api.tiles.mapbox.com/mapbox-gl-js/v0.54.0/mapbox-gl.css' rel='stylesheet' />
+<style>
+  .marker {
```

Comment:
where is this css class used? can we make it more specific and maybe add some prefix, e.g. `jekyll-maps-mapbox-marker` or something like that?